superbench/benchmarks/model_benchmarks/pytorch_bert.py

```diff
@@ -176,6 +177,7 @@ def _inference_step(self, precision):
                     self._model(sample)
                     end = self._timer()
                     curr_step += 1
+                    self._log_step_time(curr_step, precision, start, end)
                     if curr_step > self._args.num_warmup:
                         # Save the step time of every training/inference step, unit is millisecond.
                         duration.append((end - start) * 1000)
```

Comment:
log an average of last n steps in duration?

Reply:
updated